doc/ddu.txt

```diff
@@ -1298,15 +1298,20 @@ CREATE UI                                                      *ddu-create-ui*
 
 To create UI, you should read other UIs implementation.
 
-The UIs must put under "denops/@ddu-uis/*.ts".
+The UIs must be placed in one of the following structures, where {name} is
+the UI name:
+
+	denops/@ddu-uis/{name}.ts
+	denops/@ddu-uis/{name}/main.ts
+
+NOTE: If both structures exist, "{name}.ts" is preferred.
```

Comment:
There is no description in the documentation that the "deno.json[c]" and "import_map.json[c]" files are also loaded.

Reply:
Created new section `Create extension` and improved and simplify this section.